lambdas/update_supporter_appointment_settings.py

```diff
@@ -1,8 +1,8 @@
 #Written by Nhan Le
 
+from package.query_db import query
+from package.dictionary_to_list import dictionary_to_list
 import json
-import boto3
-import constants
 
```

Comment:
Add required inputs and expected outputs as commented lines before your function.